debian/postinst

```diff
@@ -1,45 +1,46 @@
 #!/bin/bash
 set -e
 
-NAME=emby-server
+NAME=jellyfin
 CONF_FILE=/etc/${NAME}.conf
 DEFAULT_FILE=/etc/default/${NAME}
 
-# Source Emby server default configuration
+# Source Jellyfin default configuration
 if [[ -f $DEFAULT_FILE ]]; then
   . $DEFAULT_FILE
 fi
 
-# Source Emby server user configuration overrides
+# Source Jellyfin user configuration overrides
 if [[ -f $CONF_FILE ]]; then
   . $CONF_FILE
 fi
 
-# Data directory where Emby database, cache and logs are stored
-PROGRAMDATA=${EMBY_DATA-/var/lib/$NAME}
+# Data directory where Jellyfin database, cache and logs are stored
+PROGRAMDATA=${JELLYFIN_DATA-/var/lib/$NAME}
 
 case "$1" in
   configure)
-    # create emby group if it does not exist
-    if [[ -z "$(getent group emby)" ]]; then
-      addgroup --quiet --system emby > /dev/null 2>&1
+    # create jellyfin group if it does not exist
+    if [[ -z "$(getent group jellyfin)" ]]; then
+      addgroup --quiet --system jellyfin > /dev/null 2>&1
     fi
-    # create emby user if it does not exist
-    if [[ -z "$(getent passwd emby)"  ]]; then
-      adduser --system --ingroup emby --shell /bin/false emby --no-create-home --home ${PROGRAMDATA} \
-        --gecos "Emby Server default user" > /dev/null 2>&1
+    # create jellyfin user if it does not exist
+    if [[ -z "$(getent passwd jellyfin)"  ]]; then
+      adduser --system --ingroup jellyfin --shell /bin/false jellyfin --no-create-home --home ${PROGRAMDATA} \
+        --gecos "Jellyfin default user" > /dev/null 2>&1
     fi
     # ensure $PROGRAMDATA has appropriate permissions
     if [[ ! -d $PROGRAMDATA ]]; then
       mkdir $PROGRAMDATA
-      chown -R emby:emby $PROGRAMDATA
+      chown -R jellyfin:jellyfin $PROGRAMDATA
     fi
-    # ensure emby-server binary has appropriate permissions
-    chmod 755 /usr/bin/emby-server
+    # ensure jellyfin binary has appropriate permissions
+    chmod 755 /usr/bin/jellyfin
 
-    /usr/bin/mono --aot=full -O=all $EMBY_BIN > /dev/null 2>&1 || true
+    chmod +x ${JELLYFIN_DIR}/restart.sh > /dev/null 2>&1 || true
 
-    chmod +x ${EMBY_DIR}/restart.sh > /dev/null 2>&1 || true
+    # Link hardcoded /var/lib/emby to /var/lib/jellyfin (TEMPORARY)
+    ln -sf /var/lib/jellyfin /var/lib/emby
```

Comment:
This would fail if emby was previously installed, is it handled by dpkg?

Reply:
Yes, I added a `Conflicts` entry for `emby-server`.